app/assets/stylesheets/application.css

```diff
@@ -14,3 +14,18 @@
  *= require dataTables/bootstrap/3/jquery.dataTables.bootstrap
  *= require_self
  */
+
+/*### START DMc - A toe-dip (first OD2 pull request 4/16/19) of calming CSS - and might be more appropriate elsewhere and as a mixin eventually ###*/
```

Comment:
CSS looks good, but typically we don't want to include these directly in `application.css`, but to create a separate .scss file which should automagically be included as part of the `*= require_tree .` in this file.

Reply:
FWIW, it might be nice to have stylesheets sort of logically divided in some way to keep related features/pages nearby each other.. there isn't any harm in having several small stylesheets if it helps the readability and maintainability

Reply:
Awesome.